### PR TITLE
Passt

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
Move modified passt kubevirt binding pluging https://github.com/qinqon/kubevirt-passt-binding at this repo as a submodule.

**Special notes for your reviewer**:
The passt binding dependencies are way bigger than the kubevirt-ipam-controller since the passt binding depends on a lot of kubevirt pkgs.

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
Passt binding
```

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -3,6 +3,6 @@ resources:
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
-- name: controller
-  newName: ghcr.io/kubevirt/ipam-controller
-  newTag: latest
+- digest: sha256:36920f15205ef980969aacfd3e956d2e6370fe22e8c3641c8a57552f1b95c3ee
+  name: controller
+  newName: localhost:5000/kubevirt-ipam-controller
```
Comment (oshoval) [inline config/manager/kustomization.yaml:8]: this might break git actions and so on, this is updated when doing cluster-sync

**Comment by oshoval** [comment]
to vendor kubevirt we just need a function like this 
https://github.com/kubevirt/ipam-extensions/blob/ef0ec97ff9c022f93096def80c374a356c0dbde6/hack/cluster.sh#L23

**Comment by qinqon** [comment]
Closing this, this since the approach is going to be vendoring kubevirt for now.

**Pull request closed without merging**